CRM/Upgrade/Incremental/php/FiveTwentyOne.php

```diff
@@ -65,8 +65,32 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
   //    // The above is an exception because 'Upgrade DB to %1: SQL' is generic & reusable.
   //  }
 
-  // public static function taskFoo(CRM_Queue_TaskContext $ctx, ...) {
-  //   return TRUE;
-  // }
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_5_21_alpha1($rev) {
+    $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
+    $this->addTask('dev/core#1405 Fix option group names that contain spaces', 'fixOptionGroupName');
+  }
+
+  public static function fixOptionGroupName() {
+    $optionGroups = \Civi\Api4\OptionGroup::get()
+      ->setCheckPermissions(FALSE)
+      ->execute();
+    foreach ($optionGroups as $optionGroup) {
+      $name = trim($optionGroup['name']);
+      if (strpos($name, ' ') !== FALSE) {
+        $fixedName = CRM_Utils_String::titleToVar($name);
```

Comment:
Convert to lowercase here as well during the upgrade.

$fixedName = CRM_Utils_String::titleToVar(strtolower($name));

Probably should at least log what option group names have been changed so that at least CiviCRM developers are forewarned after the upgrade that potentially any custom integration referring to the option group by name will now be broken. Just a suggestion.